raiden-dapp/src/store.ts

```diff
@@ -83,24 +85,27 @@ const store: StoreOptions<RootState> = {
         return acc;
       };
 
-      return map(flatMap(state.channels), tokenChannels => {
-        const model = reduce(tokenChannels, reducer, emptyTokenModel());
-        const tokenInfo = state.tokens[model.address];
-        if (tokenInfo) {
-          model.name = tokenInfo.name || '';
-          model.symbol = tokenInfo.symbol || '';
-        }
+      return map(
+        filter(flatMap(state.channels), channels => !isEmpty(channels)),
```

Comment:
IMO we should be using [flatMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap), [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) and [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) here instead of importing from `lodash`